### DISCUSSION TITLE
Introduction: Olufemi Olabode (OLA)

### DISCUSSION THREAD

**Discussion by oolu4236** [life-itself/community#970] [Introductions]
Greeting All, 
 Very glad to join this conversation. I currently live in Lagos, Nigeria where I explore meta-level concepts in my context. I have different technical skills and competencies. My approach to work integrates a wide array of creative skills into my work, spanning visual design, coding, storytelling, and a good sense of humor. I've also worn many hats, serving as a facilitator, technologist, coach, contributor, and all those meaningful roles that create "aha moments" in digital spaces.
 
I'm energized by mutually enriching online-offline relationships and constantly on the search for different ways to learn, grow, collaborate and develop with others in different capacities. 

I'm looking forward to meet you all

[Website](https://olaolu.my.canva.site/hireme)

**Comment by rufuspollock** [comment]
@oolu4236 welcome and thanks for introducing yourself 😄
